Client/Frontend/Browser/TabsBar/TabsBarViewController.swift

```diff
@@ -136,14 +140,6 @@ class TabsBarViewController: UIViewController {
         overflowIndicators()
     }
     
-    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
Theming code is called from BVC from the same method, i removed it to not apply theme twice